language-server/modules/langserver-core/src/test/resources/codeaction/type-guard/config/typeGuardVariableCodeAction2.json

```diff
@@ -3,22 +3,7 @@
     "character": 35,
     "expected": [
         {
-            "title": "Type guard variable 'val'",
-            "edits": [
-                {
-                    "range": {
-                        "start": {
-                            "line": 7,
-                            "character": 45
-                        },
-                        "end": {
-                            "line": 7,
-                            "character": 45
-                        }
-                    },
-                    "newText": "\n    if val is string {\n\n    } else if val is int {\n\n    } else {\n\n    }\n"
-                }
-            ]
+            "title": "Type guard variable 'val'"
```

Comment:
Why do we expect the title only?

Reply:
It's a negative test.

Reply:
If this is a negative case, we should not get the title also, right? Shouldn't we expect just the empty body?